docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -27,9 +27,8 @@ Linux operating system
 Java runtime environment
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-Trino requires a 64-bit version of Java 11, with a minimum required version of 11.0.7.
-Newer patch versions such as 11.0.8 or 11.0.9 are recommended. Earlier patch versions
-such as 11.0.2 do not work, nor will earlier major versions such as Java 8. Newer major
+Trino requires a 64-bit version of Java 11, with a minimum required version of 11.0.11.
+Earlier patch versions such as 11.0.2 do not work, nor will earlier major versions such as Java 8. Newer major
```

Comment:
Nit: let's put the "Newer major" part on the next line, since that looks a bit cleaner (and the line lengths match up)